test-utils/mongo-helper.js

```diff
@@ -37,7 +37,12 @@ class MongoHelper {
 
   async dropAllCollections() {
     const collections = await this.db.listCollections().toArray();
-    return Promise.all(collections.map(({ name }) => this.db.collection(name).drop()));
+    return Promise.all(collections.map(({ name }) => {
+      // System collections are not droppable…
+      if (name.startsWith('system.')) return null;
+      // …other collections are.
+      return this.db.collection(name).drop();
+    }));
```

Comment:
what about 
```
collections
   .filter(({}) => !name.startsWith('system.'))
   .map(...)
```

Reply:
Did you saw an issue fixed by this, like a strange behavior because system objects were dropped?

Reply:
> Did you saw an issue fixed by this, like a strange behavior because system objects were dropped?

Yes, see description above: https://github.com/ForestAdmin/lumber/pull/487#issue-516178015

Reply:
> what about
> 
> ```
> collections
>    .filter(({}) => !name.startsWith('system.'))
>    .map(...)
> ```

Way better! Thanks 🙏 (fixed)